internal/mirror/mirror.go

```diff
@@ -73,6 +75,19 @@ func NewMirror(targetURL string, config *config.Config, failureCh chan<- string,
 }
 
 func (m *Mirror) Reflect(req *Request) {
+	m.sendQueue.AddToQueue(req, m.targetURL)
+	// Attempt sending the next items
+	m.tryExecuteNext()
+}
+
+// This function assumes the requestsLock is held
```

Comment:
Does this refer to the mutex in the `Reflector`? I do see a Mutex in the `Mirror` struct too, but it seems to be unused...

Reply:
Was a spurious comment